olive/cli/export_adapters.py

```diff
@@ -81,6 +81,7 @@ def register_subcommand(parser: ArgumentParser):
             choices=["symmetric", "asymmetric"],
             help="Quantization mode for int4 quantization. Default is symmetric.",
         )
+        add_logging_options(sub_parser)
```

Comment:
This cli is not running Olive workflow so logging option is not valid here.

Reply:
We could log info from the CLI itself.

Reply:
We use `print` instead of log in cli. This option is used for Olive log config: https://github.com/microsoft/Olive/blob/main/olive/cli/capture_onnx.py#L174